sdk/appconfiguration/arm-appconfiguration/LICENSE.txt

```diff
@@ -1,21 +0,0 @@
-The MIT License (MIT)
```

Comment:
is the deletion intended?

Reply:
Yes because we had two license files, the .txt is out of date versus the `LICENSE` file.